workers/unity/Packages/io.improbable.gdk.core/Worker/SerializedMessagesToSend.cs

```diff
@@ -230,7 +226,7 @@ public void AddRequest(CommandRequest request, uint commandId, long entityId, ui
         public void AddResponse(CommandResponse response, uint requestId)
         {
             responses.Add(new ResponseToSend(response, requestId));
-            netFrameStats.AddCommandResponse(response, message: null);
+            netFrameStats.AddCommandResponse(response, null);
```

Comment:
Any reason to remove this? It seems a bit more clear what's being passed null with the message signifier.

Reply:
Seems the formatter got rid of this, will put it back.